src/index.js

```diff
@@ -107,12 +107,20 @@ function ClientApi(options) {
   this.websocket = new Websocket(this);
 }
 
-ClientApi.prototype.requestNew = async (url, data) => {
+ClientApi.prototype.requestNew = async (url, data, format) => {
   try {
     if (data) {
       requestConfig.params = { ...requestConfig.params, ...data };
```

Comment:
Shouldn't this use a new `requestConfig`?

Reply:
This is using new request config, if you look closely at line 117-119 I am using spread operator to override/temporarily update the value of baseURL before calling

Reply:
Isn't it assigning to existing `requestConfig`?

Reply:
I know what it's trying to do but it seems to me that it leads to pollution of parameters between API calls, parameters set in previous API calls would be included in new ones.